subsys/bluetooth/services/Kconfig

```diff
@@ -4,7 +4,11 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-comment "BLE GATT Services"
+config BT_NORDIC_SERVICES
```

Comment:
BT_NRF_SERVICES ?

Reply:
Yes, better, will update for MESH as well.